jdi-light/src/main/java/com/epam/jdi/light/driver/WebDriverByUtils.java

```diff
@@ -149,8 +161,9 @@ public static By getByFromString(String stringLocator) {
     private static List<WebElement> getEls(Object step, SearchContext ctx, List<WebElement> els) {
         if (isClass(step.getClass(), By.class)) {
             String byName = getByName((By) step);
-            if (byName.equals("id") || (byName.equals("css") && getByLocator((By) step).matches("^#[a-zA-Z-]+$"))) {
-                return getDriver().findElements((By) step);
+            if (byName.equals("id") ||
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Position literals first in String comparisons](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5482899)